.github/workflows/release.yml

```
@@ -0,0 +1,23 @@
+# This workflow will do a release after merge to develop branch
+
+name: Relese Tao extension
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types: [closed]
+jobs:
+  auto-release:
+    if: github.event.pull_request.merged == true
+    name: Automated Tao extension release
+    runs-on: ubuntu-latest
```

Use fixed version to prevent surprises.

```suggestion
    runs-on: ubuntu-20.04
```